easybuild/easyconfigs/s/shovill/shovill-1.1.0-gompi-2021b.eb

```
@@ -33,7 +33,7 @@ dependencies = [
 ]
 
 postinstallcmds = [
-    """echo exec java -jar \$EBROOTTRIMMOMATIC/trimmomatic-*.jar '"$@"' > %(installdir)s/bin/trimmomatic""",
+    """echo exec java -jar \\$EBROOTTRIMMOMATIC/trimmomatic-*.jar '"$@"' > %(installdir)s/bin/trimmomatic""",
```

consistency across the `shovill` easyconfigs
```suggestion
    r"""echo exec java -jar \$EBROOTTRIMMOMATIC/trimmomatic-*.jar '"$@"' > %(installdir)s/bin/trimmomatic""",
```